### ISSUE TITLE
Formatting emails

### ISSUE THREAD

**Issue by valeriupredoi** [valeriupredoi/valeriupredoi.github.io#13]
@SophieMacadamia please don't format emails like this `[Joe Doe](<joe.doe@mef.com>)` because the email is not a http link so redirecting via shortcut doesn't work. If you need to add emails just add them simply straight like `<joe.doe@mef.com>` :beer: I've corrected (I think) all those instances, but if you find another one pls correct it :+1:

**Comment by SophieMacadamia** [comment]
Thanks ! About that, on the ENES Portal, I saw that those email are followed by :?/subject=information about CMCC-ESM2 (or another object like that), meaning that when those people receive an email, it automatically generates a specific object. Do you think it is possible on jekyll?

**Comment by SophieMacadamia** [comment]
I noticed that vy doing the <emailaddress> directly, you cannot "hide" it. Like for e.g. write "Sophie Morellon" and people are led to my email address when they click on it does not seem possible. I guess that's one of Jekyll's contraints. 
The only thing that bothers me with that is that automatic bots can pick up any address (that is why on the website/portal, we used to "hide" them by writing <emailadress[a]domain.fr> for eg...
